lib/Db/AclMapper.php

```diff
@@ -29,22 +29,53 @@
 use OCP\IDBConnection;
 
 class AclMapper extends DeckMapper implements IPermissionMapper {
+
+	/**
+	 * @param IDBConnection $db
```

Comment:
I'd vote for dropping phpdoc comments if methods are typed already ;)

Reply:
Removed doc blocks for constructors, but kept them for other signed functions since I think think they are still handy for exception propagation. Any thoughts?